managers/neo4j.go

```diff
@@ -105,12 +101,27 @@ func mapRecordsToUserSummaries(records []*neo4j.Record) ([]*protos.UserSummary,
 			return nil, errors.New("expected key userNode to be found in record")
 		}
 		userNode := v.(map[string]interface{})
+
+		var Photos = make([]string, 0)
```

Comment:
Since this is a local variable, you don't need to capitalize it.
```suggestion
		photos := make([]string, 0)
```
Same applies for `Labels` and `Filters`.